lib/chef/resource/aws_security_group.rb

```diff
@@ -7,11 +7,45 @@ class Chef::Resource::AwsSecurityGroup < Chef::Provisioning::AWSDriver::AWSResou
   actions :create, :delete, :nothing
   default_action :create
 
-  attribute :name,          kind_of: String, name_attribute: true
-  attribute :vpc,           kind_of: [ String, AwsVpc, AWS::EC2::VPC ]
-  attribute :description,   kind_of: String
-  attribute :inbound_rules
-  attribute :outbound_rules
+  attribute :name,           kind_of: String, name_attribute: true
+  attribute :vpc,            kind_of: [ String, AwsVpc, AWS::EC2::VPC ]
```

Comment:
I'd actually rather not, because on *update* you don't have to care what VPC it's in.  We should just error out on create if you don't specify one.